Droidcon-Boston/app/src/main/res/layout/agenda_detail_fragment.xml

```diff
@@ -133,6 +133,39 @@
                 android:textColor="@android:color/black"
 
                 tools:text="@tools:sample/lorem/random" />
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                android:layout_below="@id/tv_agenda_detail_description"
+                android:layout_marginTop="16dp"
+                >
+
+                <TextView
+                    android:id="@+id/tv_agenda_detail_shareText"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textAlignment="center"
+                    android:textColor="@android:color/black"
+                    android:fontFamily="sans-serif"
+                    android:textSize="@dimen/def_padding_large"
+                    android:layout_centerVertical="true"
+                    tools:text="Enjoy this talk? Let your followers know!"/>
+
+                <ImageView
+                    android:id="@+id/imgv_twitter"
+                    android:layout_width="wrap_content"
+                    android:layout_height="40dp"
+                    android:contentDescription="@string/twitter_icon"
+                    android:paddingStart="@dimen/def_padding"
```

Comment:
we could convert this to a single android:Padding: ... since they are all the same